docs/attachments.md

```diff
@@ -53,20 +82,40 @@ The uploadBatchFile operation creates a batch of attachment records.
 
 **Sample request**
 
-Following is a sample request that can be handled by the uploadBatchFile operation.
+Following is a sample request that can be handled by the uploadBatchFile operation and attach the file as form data.
```

Comment:
Following is a sample request that can be handled by the uploadBatchFile operation. It creates a job for batches contain attachment records.

Reply:
Fixed in d9f4b3d6a3195a5dabee1246154595491561aa9b.